index.bs

```diff
@@ -457,7 +457,7 @@ co-references is now limited to `ex:myThirdSurface`.
 ## Positive Surface ## {#PositiveSurface}
 
 A positive surface is an RDF Graph which claims that an RDF Graph on it is true.
```

Comment:
I tripped on this when I made my #11. `An RDF surface is a kind of sheet of paper on which RDF graphs can be written` but `A positive surface is an RDF Graph which claims that an RDF Graph on it is true.` I think that `A positive surface` and `A negative surface` should be defined as `... a surface which claims ...` not `... an RDF Graph which claims ...`
```suggestion
A positive surface is a surface which claims that an RDF Graph on it is true.
```
Negative surface hasn't (yet) been touched in this PR, so I can't make that suggestion directly here; still, I think it would be quicker and easier for @joachimvh to add that change to this PR than for me to make a new PR with these two changes. I suggested one above, hopefully @joachimvh can quickly make the other.